fedora_distro_aliases/cli.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import sys
+
+from fedora_distro_aliases import get_distro_aliases
+
+GETTERS = {
+    "namever": lambda x: x.namever,
+    "branch": lambda x: x.branch,
+    "version": lambda x: x.version,
+}
+
+
+def run(output_type: str, aliases: list[str]):
+    getter = GETTERS[output_type]
+    distro_aliases = get_distro_aliases()
+
+    resolved_aliases: set[str] = set()
+    for alias in aliases:
+        resolved_aliases.update(getter(x) for x in distro_aliases[alias])
+
+    print(" ".join(resolved_aliases))
+
+
+def setup_parser() -> argparse.ArgumentParser:
```

Comment:
Nice, this should allow you to generate manual pages for free:
$ argparse-manpage --pyfile fedora_distro_aliases/cli.py  --function setup_parser > resolve-fedora-aliases.1

Reply:
One thing I noticed is that `argparse` ignores newlines, now that you mention the manpages, it feels like a bit poor decision on their end :thinking:

Reply:
Yup, you need to use the raw formatter: https://github.com/rpm-software-management/mock/pull/1393/files#diff-502fd2d3cb5ca4a464155aec5622d19895ccc608512f9334095f08c2f9e41947R65

Reply:
I'd create the manpage in a follow-up PR, as this has been open for some time already, would that be an issue?

Reply:
No no, a follow-up PR is fine :-)
Thank you